src/arm/windows/init-by-logical-sys-info.c

```diff
@@ -832,8 +832,9 @@ static bool connect_packages_cores_clusters_by_processors(
 		processor->cluster = cluster;
 
 		if (chip_info) {
-			strncpy_s(package->name, CPUINFO_PACKAGE_NAME_MAX, chip_info->chip_name_string,
-				strnlen(chip_info->chip_name_string, CPUINFO_PACKAGE_NAME_MAX));
+			size_t converted_chars = 0;
+			wcstombs_s(&converted_chars, package->name, CPUINFO_PACKAGE_NAME_MAX,

```

Comment:
I prefer WinAPI calls over CRT, as cpuinfo may be compiled with non-Microsoft compilers (e.g. Cygwin). Please use `WideCharToMultiByte` instead of `wcstombs_s`.

Reply:
Hi, changed to use `WideCharToMultiByte`: I think I've done it correctly and it seems to pass the tests (in Unicode and non-Unicode variants), but I think you're familiar with these things than I am. Could you take a look at `2b261b9`?